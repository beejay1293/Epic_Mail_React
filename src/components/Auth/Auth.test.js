import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Form from './auth';
describe('Auth', () => {
  it('should render correctly in "debug" mode', () => {
    const Auth = shallow(<Form />);
  
    expect(Auth).toMatchSnapshot();
  });
});