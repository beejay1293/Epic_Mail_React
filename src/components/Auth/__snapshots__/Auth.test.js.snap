// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth should render correctly in "debug" mode 1`] = `
<div
  className="container"
>
  <div
    className="login__wrapper"
  >
    <h1
      className="header"
    >
      <Link
        className="link"
        to="/"
      >
        <span>
          <em>
            E
          </em>
        </span>
        PIC
        <b
          className="mails"
        >
          Mail
        </b>
      </Link>
    </h1>
    <Forms />
  </div>
  <div
    className="wrapper"
  />
</div>
`;
